cheddar/cheddar-metrics-intercom/src/main/java/com/clicktravel/cheddar/metrics/intercom/IntercomMetricCollector.java

```diff
@@ -69,6 +69,15 @@ public void updateUser(final MetricUser user) {
             logger.debug("Error updating a Intercom user: " + intercomUser + " - " + e.getMessage());
         }
     }
+    
+    @Override
+    public void deleteUser(final String userId) {
+    	try {
```

Comment:
I thought that it's not necessary to have an `MetricUser` object as parameter, as only user's id is needed. Do you think it's better to have consistency across all methods?